e2e/next-sandbox/test/comments/index.test.ts

```diff
@@ -0,0 +1,42 @@
+import type { Page } from "@playwright/test";
+import { test } from "@playwright/test";
+
+import {
+  genRoomId,
+  getJson,
+  preparePages,
+  waitForJson,
+  waitUntilEqualOnAllPages,
+} from "../utils";
+
+const TEST_URL = "http://localhost:3007/comments";
+
+test.describe("Threads", () => {
+  let pages: [Page, Page];
+
+  test.beforeEach(async ({}, testInfo) => {
+    const room = genRoomId(testInfo);
+    pages = await preparePages(`${TEST_URL}?room=${encodeURIComponent(room)}`);
+  });
+
+  test.afterEach(() => Promise.all(pages.map((page) => page.close())));
+
+  test("verify A and B display same number of threads after threads are loaded", async () => {
+    await waitForJson(pages, "#isLoading", false);
+    await waitUntilEqualOnAllPages(pages, "#numOfThreads");
```

Comment:
This tests for the initial loading of thread and verifies that threads are loaded correctly and all clients see the same number of threads.

Reply:
When I try to run it locally (against the DEV servers), I'm seeing this error message:

<img width="699" alt="Screen Shot 2023-10-13 at 17 35 20@2x" src="https://github.com/liveblocks/liveblocks/assets/83844/bf9ec791-4a91-46ee-bcd9-ae1151cea95d">

Reply:
That is probably happening because,`serverEndpoint` need to be updated to point at the dev comments server while testing in dev.

https://github.com/liveblocks/liveblocks/blob/main/packages/liveblocks-core/src/room.ts#L2256-L2258

```ts
  const commentsApi = createCommentsApi(config.roomId, delegates.authenticate, {
    serverEndpoint: "https://api.liveblocks.io/v2",
  });
```

Reply:
Ohh, good that you mention this. We were thinking to unify our various env var configs (client, comments, node) into just one new config var named `baseUrl`, which would replace `liveblocksServer`, `liveblocksAuthorizeEndpoint`, `liveblocksBaseUrl`, and maybe also this `serverEndpoint`. The idea is that we strip the `/v2` part from this config value, and that all three existing variables could then be _derived_ from the one `baseUrl` config then.

Any objections against me doing this soon, @marcbouchenoire @nimeshnayaju ?

Reply:
I created a ticket for this one: https://github.com/liveblocks/liveblocks/issues/1230